### ISSUE TITLE
File explorer doesn't save sort preferences 

### ISSUE THREAD

**Issue by ghost** [Vanilla-OS/iso-builder#128]
### Issue Description

File explorer doesn't save filter preferences in the files' app. Such as sort by date, name, etc

### Steps to Reproduce

- `open` files 
- `sort` folders by `latest date modified` 
- `enter` an folder
- `exit` the folder

You will notice it didn't save

### On what version of Vanilla OS this happens?

22.10

### Additional Information

[Screencast from 2023-02-08 08-52-32.webm](https://user-images.githubusercontent.com/122128828/217549369-f6dcd8d2-0b22-4a7c-a2d3-3b01fe844a3f.webm)

## 22.10 R6

**Comment by kbdharun** [comment]
I am in a fresh install and I can't replicate this issue 🤔.

**Comment by ghost** [comment]
> I am in a fresh install and I can't replicate this issue thinking.

It seems to be reoccurring for me, this is my 2nd restart

**Comment by kbdharun** [comment]
> > I am in a fresh install and I can't replicate this issue thinking.
> 
> It seems to be reoccurring for me, this is my 2nd restart

No idea, tbh what causes this issue.

**Comment by ghost** [comment]
> > > I am in a fresh install and I can't replicate this issue thinking.
> > 
> > 
> > It seems to be reoccurring for me, this is my 2nd restart
> 
> No idea, tbh what causes this issue.
I have no idea either here is my log messages 

~~~
10:50:52 nautilus: Called "net usershare info" but it failed: 'net usershare' returned error 255: net usershare: cannot open usershare directory /var/lib/samba/usershares. Error Permission denied
You do not have permission to create a usershare. Ask your administrator to grant you permissions to create a share.

10:50:52 systemd: Started Hostname Service.
10:50:52 dbus-daemon: [system] Successfully activated service 'org.freedesktop.hostname1'
10:50:51 gnome-shell: meta_window_set_stack_position_no_sync: assertion 'window->stack_position >= 0' failed
10:50:51 systemd: Starting Hostname Service...
10:50:51 dbus-daemon: [system] Activating via systemd: service name='org.freedesktop.hostname1' unit='dbus-org.freedesktop.hostname1.service' requested by ':1.215' (uid=1000 pid=12458 comm="/usr/bin/nautilus --gapplication-service" label="unconfined")
10:50:51 gnome-photos: Unable to fetch the list of available renderers: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name com.intel.dleyna-renderer was not provided by any .service files
10:50:51 gnome-shell: Received error from D-Bus search provider org.gnome.Terminal.desktop: Gio.DBusError: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Object does not exist at path “/org/gnome/Terminal/SearchProvider”
10:50:51 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.Photos'
10:50:51 gnome-shell: Received error from D-Bus search provider org.gnome.Terminal.desktop: Gio.DBusError: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Object does not exist at path “/org/gnome/Terminal/SearchProvider”
10:50:51 dbus-daemon: [session uid=1000 pid=1042] Activating service name='org.gnome.Photos' requested by ':1.30' (uid=1000 pid=1190 comm="/usr/bin/gnome-shell" label="unconfined")
10:50:45 gnome-shell: meta_window_set_stack_position_no_sync: assertion 'window->stack_position >= 0' failed
10:50:44 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.Logs'
10:50:44 systemd: Started app-flatpak-org.gnome.Logs-12534.scope.
10:50:43 dbus-daemon: [session uid=1000 pid=1042] Activating service name='org.gnome.Logs' requested by ':1.30' (uid=1000 pid=1190 comm="/usr/bin/gnome-shell" label="unconfined")
10:50:43 gnome-photos: Unable to fetch the list of available renderers: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name com.intel.dleyna-renderer was not provided by any .service files
10:50:43 gnome-shell: Received error from D-Bus search provider org.gnome.Terminal.desktop: Gio.DBusError: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Object does not exist at path “/org/gnome/Terminal/SearchProvider”
10:50:43 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.Console'
10:50:43 nautilus: Connecting to org.freedesktop.Tracker3.Miner.Files
10:50:43 gnome-shell: Received error from D-Bus search provider org.gnome.Terminal.desktop: Gio.DBusError: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Object does not exist at path “/org/gnome/Terminal/SearchProvider”
10:50:43 systemd: Started GNOME Terminal Server.
10:50:43 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.Terminal'
10:50:43 systemd: Starting GNOME Terminal Server...
10:50:43 dbus-daemon: [session uid=1000 pid=1042] Activating via systemd: service name='org.gnome.Terminal' unit='gnome-terminal-server.service' requested by ':1.30' (uid=1000 pid=1190 comm="/usr/bin/gnome-shell" label="unconfined")
10:50:39 systemd: fprintd.service: Deactivated successfully.
10:50:32 lbry: Daemon: 2023-02-08 10:50:32,204 INFO     lbry.extras.daemon.daemon:571: finished shutting down
10:50:30 systemd: NetworkManager-dispatcher.service: Deactivated successfully.
10:50:25 firefox-bin: ATTENTION: default value of option mesa_glthread overridden by environment.
10:50:21 systemd: Started app-flatpak-org.mozilla.firefox-11957.scope.
10:50:21 cron: pam_unix(cron:session): session closed for user root
10:50:21 gnome-photos: Unable to fetch the list of available renderers: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name com.intel.dleyna-renderer was not provided by any .service files
10:50:21 gnome-shell: Timelines with detached actors are not supported. <unnamed>[<Gjs_ui_search_ListSearchResult>:0x558c16046350] in animation of duration 200ms but not on stage.
10:50:21 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.Photos'
10:50:21 gnome-shell: Received error from D-Bus search provider org.gnome.Terminal.desktop: Gio.DBusError: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Object does not exist at path “/org/gnome/Terminal/SearchProvider”
10:50:21 dbus-daemon: [session uid=1000 pid=1042] Activating service name='org.gnome.Photos' requested by ':1.30' (uid=1000 pid=1190 comm="/usr/bin/gnome-shell" label="unconfined")
10:50:20 NetworkManager: <info>  [1675871420.5820] device (tun0): Activation: successful, device activated.
10:50:20 systemd-resolve: tun0: Bus client set DNS server list to: 10.81.0.1
10:50:20 NetworkManager: <info>  [1675871420.5560] device (tun0): state change: config -> ip-config (reason 'none', sys-iface-state: 'external')
10:50:20 charon: 04[KNL] 10.81.0.10 appeared on tun0
10:50:20 openvpn: Initialization Sequence Completed
10:50:20 charon: 14[KNL] fe80::df81:eab3:aa34:c4d8 appeared on tun0
10:50:20 NetworkManager: <info>  [1675871420.5320] manager: (tun0): new Tun device (/org/freedesktop/NetworkManager/Devices/7)
10:50:20 openvpn: /usr/lib/NetworkManager/nm-openvpn-service-openvpn-helper --debug 0 11890 --bus-name org.freedesktop.NetworkManager.openvpn.Connection_5 --tun -- tun0 1500 0 10.81.0.10 255.255.0.0 init
10:50:19 charon: 04[KNL] flags changed for fe80::9633:70c0:8621:7224 on wlp3s0
10:50:19 openvpn: [node-us-46.protonvpn.net] Peer Connection Initiated with [AF_INET]193.148.18.34:7770
10:50:18 NetworkManager: <info>  [1675871418.4909] audit: op="connection-activate" uuid="e7c2480a-d0f6-469a-87e4-307fecbf430c" name="US-NY#20 (P2P)" pid=1190 uid=1000 result="success"
10:50:17 geoclue: Failed to query location: No WiFi networks found
10:50:17 NetworkManager: <info>  [1675871417.9421] manager: NetworkManager state is now CONNECTED_GLOBAL
10:50:17 systemd: Started Network Manager Script Dispatcher Service.
10:50:17 dbus-daemon: [system] Successfully activated service 'org.freedesktop.nm_dispatcher'
10:50:17 systemd: Starting Network Manager Script Dispatcher Service...
10:50:17 dbus-daemon: [system] Activating via systemd: service name='org.freedesktop.nm_dispatcher' unit='dbus-org.freedesktop.nm-dispatcher.service' requested by ':1.12' (uid=0 pid=820 comm="/usr/sbin/NetworkManager --no-daemon" label="unconfined")
10:50:17 NetworkManager: <info>  [1675871417.6949] device (wlp3s0): state change: ip-config -> ip-check (reason 'none', sys-iface-state: 'managed')
10:50:17 avahi-daemon: Registering new address record for 10.1.227.205 on wlp3s0.IPv4.
10:50:17 systemd-resolve: wlp3s0: Bus client set DNS server list to: 10.84.2.7, 8.8.8.8, 8.8.4.4
10:50:17 charon: 14[KNL] 10.1.227.205 appeared on wlp3s0
10:50:17 avahi-daemon: Joining mDNS multicast group on interface wlp3s0.IPv4 with address 10.1.227.205.
10:50:17 NetworkManager: <info>  [1675871417.6112] policy: set 'CU-Guest' (wlp3s0) as default for IPv4 routing and DNS
10:50:17 avahi-daemon: Registering new address record for fe80::9633:70c0:8621:7224 on wlp3s0.*.
10:50:17 charon: 13[KNL] fe80::9633:70c0:8621:7224 appeared on wlp3s0
10:50:17 avahi-daemon: Joining mDNS multicast group on interface wlp3s0.IPv6 with address fe80::9633:70c0:8621:7224.
10:50:17 NetworkManager: <info>  [1675871417.5977] dhcp4 (wlp3s0): activation: beginning transaction (timeout in 45 seconds)
10:50:17 wpa_supplicant: wlp3s0: CTRL-EVENT-SIGNAL-CHANGE above=1 signal=-53 noise=9999 txrate=117000
10:50:17 kernel: IPv6: ADDRCONF(NETDEV_CHANGE): wlp3s0: link becomes ready
10:50:17 wpa_supplicant: wlp3s0: CTRL-EVENT-CONNECTED - Connection to d0:d3:e0:60:f0:f2 completed [id=0 id_str=]
10:50:17 NetworkManager: <info>  [1675871417.5435] device (p2p-dev-wlp3s0): supplicant management interface state: authenticating -> associating
10:50:17 wpa_supplicant: wlp3s0: Trying to associate with d0:d3:e0:60:f0:f2 (SSID='CU-Guest' freq=5785 MHz)
10:50:17 NetworkManager: <info>  [1675871417.5114] device (p2p-dev-wlp3s0): supplicant management interface state: disconnected -> authenticating
10:50:17 kernel: wlp3s0: authenticate with d0:d3:e0:60:f0:f2
10:50:17 NetworkManager: <info>  [1675871417.4841] Config: added 'key_mgmt' value 'NONE'
10:50:14 kernel: psmouse serio1: issuing reconnect request
10:50:13 gnome-photos: Unable to fetch the list of available renderers: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name com.intel.dleyna-renderer was not provided by any .service files
10:50:13 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.Console'
10:50:13 nautilus: Connecting to org.freedesktop.Tracker3.Miner.Files
10:50:13 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.Epiphany.SearchProvider'
10:50:13 gnome-shell: Received error from D-Bus search provider org.gnome.Terminal.desktop: Gio.DBusError: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Object does not exist at path “/org/gnome/Terminal/SearchProvider”
10:50:13 systemd: Started GNOME Terminal Server.
10:50:13 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.Terminal'
10:50:13 systemd: Starting GNOME Terminal Server...
10:50:13 dbus-daemon: [session uid=1000 pid=1042] Activating service name='org.gnome.Epiphany.SearchProvider' requested by ':1.30' (uid=1000 pid=1190 comm="/usr/bin/gnome-shell" label="unconfined")
10:50:12 systemd: NetworkManager-dispatcher.service: Deactivated successfully.
10:50:10 gnome-shell: Can not connect to https://openweathermap.org/city/0
Error resolving “openweathermap.org”: Name or service not known
10:50:09 NetworkManager: <info>  [1675871409.5487] agent-manager: agent[bce82993536f242d,:1.32/org.gnome.Shell.NetworkAgent/1000]: agent registered
10:50:09 gnome-shell: object string = /org/ayatana/NotificationItem/LBRY1
10:50:09 gdm-session-wor: gkr-pam: no password is available for user
10:50:08 gnome-shell: JS WARNING: [resource:///org/gnome/gjs/modules/core/overrides/Gio.js 287]: Too many arguments to method Gio.AsyncInitable.init_async: expected 3, got 4
10:50:08 systemd: Started Fingerprint Authentication Daemon.
10:50:08 dbus-daemon: [system] Successfully activated service 'net.reactivated.Fprint'
10:50:08 wpa_supplicant: wlp3s0: CTRL-EVENT-REGDOM-CHANGE init=DRIVER type=COUNTRY alpha2=US
10:50:08 systemd: Starting Fingerprint Authentication Daemon...
10:50:08 dbus-daemon: [system] Activating via systemd: service name='net.reactivated.Fprint' unit='fprintd.service' requested by ':1.32' (uid=1000 pid=1190 comm="/usr/bin/gnome-shell" label="unconfined")
10:50:07 ModemManager: <info>  [base-manager] couldn't check support for device '/sys/devices/pci0000:00/0000:00:02.3/0000:03:00.0': not supported by any plugin
10:50:05 charon: 08[KNL] interface wlp3s0 activated
10:50:05 kernel: r8169 0000:02:00.0 enp2s0f0: Link is Down
10:50:05 NetworkManager: <info>  [1675871405.3814] device (wlp3s0): state change: unmanaged -> unavailable (reason 'managed', sys-iface-state: 'external')
10:50:05 charon: 01[KNL] interface enp2s0f0 activated
10:50:05 kernel: Generic FE-GE Realtek PHY r8169-0-200:00: attached PHY driver (mii_bus:phy_addr=r8169-0-200:00, irq=MAC)
10:50:05 NetworkManager: <info>  [1675871405.2249] device (enp2s0f0): state change: unmanaged -> unavailable (reason 'managed', sys-iface-state: 'external')
10:50:05 systemd-logind: Operation 'sleep' finished.
10:50:05 systemd: Stopped target Suspend.
10:50:05 systemd-sleep: System returned from sleep state.
10:50:05 systemd-logind: Lid opened.
10:50:05 bluetoothd: Controller resume with wake event 0x0
10:50:05 systemd-resolve: Clock change detected. Flushing caches.
10:50:05 kernel: PM: suspend exit
10:50:05 kernel: thermal thermal_zone0: failed to read out thermal zone (-61)
10:50:05 kernel: Restarting tasks ... done.
10:50:05 kernel: OOM killer enabled.
10:50:05 kernel: usb 2-2: reset high-speed USB device number 2 using xhci_hcd
10:50:05 kernel: nvme nvme0: 15/0/0 default/read/poll queues
10:50:05 kernel: amdgpu 0000:06:00.0: amdgpu: ring jpeg_dec uses VM inv eng 6 on hub 1
10:50:05 kernel: [drm] JPEG decode initialized successfully.
10:50:05 kernel: amdgpu 0000:06:00.0: amdgpu: SMU is resumed successfully!
10:50:05 kernel: [drm] PTB located at 0x000000F400900000
10:50:05 kernel: xhci_hcd 0000:05:00.0: Zeroing 64bit base registers, expecting fault
10:50:05 kernel: usb usb3: root hub lost power or was reset
10:50:05 kernel: pci 0000:00:00.2: PCI INT A: no GSI
10:50:05 kernel: ACPI: EC: interrupt unblocked
10:50:05 kernel: ACPI Error: Aborting method \_SB.PEP._DSM due to previous error (AE_NOT_FOUND) (20220331/psparse-529)
10:50:05 kernel: 
10:50:05 kernel: Initialized Arguments for Method [_DSM]:  (4 arguments defined for method invocation)
10:50:05 kernel: 
10:50:05 kernel: No Local Variables are initialized for Method [_DSM]
10:50:05 kernel: 
10:50:05 kernel: ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.LPC0.EC0.HKEY.MSCB], AE_NOT_FOUND (20220331/psargs-330)
10:50:05 kernel: 
10:50:05 kernel: Initialized Arguments for Method [_DSM]:  (4 arguments defined for method invocation)
10:50:05 kernel: 
10:50:05 kernel: No Local Variables are initialized for Method [_DSM]
10:50:05 kernel: 
10:50:05 kernel: ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.LPC0.EC0.HKEY.MSCB], AE_NOT_FOUND (20220331/psargs-330)
10:50:05 kernel: 
10:50:05 kernel: Initialized Arguments for Method [_DSM]:  (4 arguments defined for method invocation)
10:50:05 kernel: 
10:50:05 kernel: No Local Variables are initialized for Method [_DSM]
10:50:05 kernel: 
10:50:05 kernel: ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.LPC0.EC0.HKEY.MSCB], AE_NOT_FOUND (20220331/psargs-330)
10:50:05 kernel: 
10:50:05 kernel: Initialized Arguments for Method [_DSM]:  (4 arguments defined for method invocation)
10:50:05 kernel: 
10:50:05 kernel: No Local Variables are initialized for Method [_DSM]
10:50:05 kernel: 
10:50:05 kernel: ACPI BIOS Error (bug): Could not resolve symbol [\_SB.PCI0.LPC0.EC0.HKEY.MSCB], AE_NOT_FOUND (20220331/psargs-330)
10:50:05 kernel: ACPI: EC: interrupt blocked
10:50:05 kernel: printk: Suspending console(s) (use no_console_suspend to debug)
10:50:05 kernel: Freezing remaining freezable tasks ... (elapsed 0.001 seconds) done.
10:50:05 kernel: OOM killer disabled.
10:50:05 kernel: Freezing user space processes ... (elapsed 0.002 seconds) done.
10:50:05 kernel: Filesystems sync: 0.371 seconds
09:43:42 systemd: Finished GRUB failed boot detection.
09:43:42 kernel: PM: suspend entry (s2idle)
09:43:42 systemd: Finished Record successful boot for GRUB.
09:43:42 systemd-sleep: Entering sleep state 'suspend'...
09:43:42 systemd: Starting System Suspend...
09:43:42 wpa_supplicant: nl80211: deinit ifname=wlp3s0 disabled_11b_rates=0
09:43:41 openvpn: SIGTERM[hard,] received, process exiting
09:43:41 NetworkManager: <info>  [1675867421.9834] device (tun0): state change: activated -> unmanaged (reason 'unmanaged', sys-iface-state: 'removed')
09:43:41 lbry: Daemon: 2023-02-08 09:43:41,960 WARNING  lbry.extras.daemon.exchange_rate_manager:94: Error trying to connect to exchange rate CoinEx: Cannot connect to host api.coinex.com:443 ssl:default [Temporary failure in name resolution]
09:43:41 wpa_supplicant: nl80211: deinit ifname=p2p-dev-wlp3s0 disabled_11b_rates=0
09:43:41 geoclue: WiFi scan failed
09:43:41 charon: 13[KNL] interface tun0 deleted
09:43:41 systemd-resolve: tun0: Bus client reset DNS server list.
09:43:41 openvpn: Linux can't del IP from iface tun0
09:43:41 systemd-resolve: tun0: Bus client set default route setting: no
09:43:41 NetworkManager: <info>  [1675867421.7955] device (wlp3s0): state change: disconnected -> unmanaged (reason 'sleeping', sys-iface-state: 'managed')
09:43:41 openvpn: event_wait : Interrupted system call (fd=-1,code=4)
09:43:41 NetworkManager: <info>  [1675867421.7935] device (wlp3s0): supplicant interface state: completed -> disconnected
09:43:41 systemd-resolve: wlp3s0: Bus client reset DNS server list.
09:43:41 charon: 10[KNL] 10.3.227.129 disappeared from wlp3s0
09:43:41 avahi-daemon: Interface wlp3s0.IPv4 no longer relevant for mDNS.
09:43:41 NetworkManager: <info>  [1675867421.6985] dhcp4 (wlp3s0): state changed no lease
09:43:41 avahi-daemon: Interface wlp3s0.IPv6 no longer relevant for mDNS.
09:43:41 NetworkManager: <info>  [1675867421.6985] dhcp4 (wlp3s0): activation: beginning transaction (timeout in 45 seconds)
09:43:41 charon: 09[KNL] fe80::6e63:3660:bd9f:2dc7 disappeared from wlp3s0
09:43:41 NetworkManager: <info>  [1675867421.6985] dhcp4 (wlp3s0): canceled DHCP transaction
09:43:41 avahi-daemon: Leaving mDNS multicast group on interface wlp3s0.IPv6 with address fe80::6e63:3660:bd9f:2dc7.
09:43:41 NetworkManager: <info>  [1675867421.6974] device (wlp3s0): state change: deactivating -> disconnected (reason 'sleeping', sys-iface-state: 'managed')
09:43:41 wpa_supplicant: wlp3s0: CTRL-EVENT-DSCP-POLICY clear_all
09:43:41 gnome-shell: Source ID 208312 was not found when attempting to remove it
09:43:41 kernel: wlp3s0: deauthenticating from e8:26:89:2e:e5:35 by local choice (Reason: 3=DEAUTH_LEAVING)
09:43:41 systemd: Started Network Manager Script Dispatcher Service.
09:43:41 NetworkManager: <info>  [1675867421.5366] device (wlp3s0): state change: activated -> deactivating (reason 'sleeping', sys-iface-state: 'managed')
09:43:41 dbus-daemon: [system] Successfully activated service 'org.freedesktop.nm_dispatcher'
09:43:41 NetworkManager: <info>  [1675867421.5356] device (p2p-dev-wlp3s0): state change: disconnected -> unmanaged (reason 'sleeping', sys-iface-state: 'managed')
09:43:41 dbus-daemon: [system] Activating via systemd: service name='org.freedesktop.nm_dispatcher' unit='dbus-org.freedesktop.nm-dispatcher.service' requested by ':1.12' (uid=0 pid=820 comm="/usr/sbin/NetworkManager --no-daemon" label="unconfined")
09:43:41 NetworkManager: <info>  [1675867421.5258] device (enp2s0f0): state change: unavailable -> unmanaged (reason 'sleeping', sys-iface-state: 'managed')
09:43:41 charon: 13[KNL] interface enp2s0f0 deactivated
09:43:41 NetworkManager: <info>  [1675867421.5256] manager: sleep: sleep requested (sleeping: no  enabled: yes)
09:43:41 systemd-logind: Suspending...
09:43:40 systemd: app-flatpak-org.signal.Signal-10673.scope: Consumed 42.523s CPU time.
09:43:38 lbry: Daemon: 2023-02-08 09:43:38,620 INFO     lbry.torrent.tracker:190: Tracker: finished announcing 17 files to tracker.lbry.com:9252, 17 errors
09:32:58 systemd: Run anacron jobs was skipped because of a failed condition check (ConditionACPower=true).
09:31:58 lbry: Daemon: 2023-02-08 09:31:58,503 INFO     lbry.torrent.tracker:190: Tracker: finished announcing 17 files to tracker.lbry.com:9252, 17 errors
09:30:58 systemd: Finished Ubuntu Advantage Timer for running repeated jobs.
09:30:48 lbry: Daemon: 2023-02-08 09:30:48,499 INFO     lbry.torrent.tracker:190: Tracker: finished announcing 17 files to tracker.lbry.com:9252, 17 errors
09:30:01 cron: pam_unix(cron:session): session closed for user root
09:30:01 kernel: (root) CMD ([ -x /etc/init.d/anacron ] && if [ ! -d /run/systemd/system ]; then /usr/sbin/invoke-rc.d anacron start >/dev/null; fi)
09:30:01 cron: pam_unix(cron:session): session opened for user root(uid=0) by (uid=0)
09:29:38 lbry: Daemon: 2023-02-08 09:29:38,496 INFO     lbry.torrent.tracker:190: Tracker: finished announcing 17 files to tracker.lbry.com:9252, 17 errors
09:26:33 cat: (Use `exe --trace-deprecation ...` to show where the warning was created)
09:26:30 systemd: Started app-flatpak-org.signal.Signal-10740.scope.
09:26:30 gnome-photos: Unable to fetch the list of available renderers: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name com.intel.dleyna-renderer was not provided by any .service files
09:26:30 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.Photos'
09:26:30 gnome-photos: Unable to fetch the list of available renderers: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name com.intel.dleyna-renderer was not provided by any .service files
09:26:30 systemd: Started flatpak portal.
09:26:30 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.freedesktop.portal.Flatpak'
09:26:30 systemd: Starting flatpak portal...
09:26:30 dbus-daemon: [session uid=1000 pid=1042] Activating via systemd: service name='org.freedesktop.portal.Flatpak' unit='flatpak-portal.service' requested by ':1.322' (uid=1000 pid=10681 comm="xdg-dbus-proxy --args=42" label="unconfined")
09:26:30 signal-desktop: Debug: Additionally, user gave:
09:26:29 systemd: Started app-flatpak-org.signal.Signal-10673.scope.
09:26:29 nautilus: Connecting to org.freedesktop.Tracker3.Miner.Files
09:26:29 systemd: Started Application launched by gnome-shell.
09:26:29 gnome-shell: Received error from D-Bus search provider org.gnome.Terminal.desktop: Gio.DBusError: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Object does not exist at path “/org/gnome/Terminal/SearchProvider”
09:26:29 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.Epiphany.SearchProvider'
09:26:29 systemd: Started GNOME Terminal Server.
09:26:29 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.Terminal'
09:26:29 systemd: Starting GNOME Terminal Server...
09:26:29 dbus-daemon: [session uid=1000 pid=1042] Activating service name='org.gnome.Epiphany.SearchProvider' requested by ':1.30' (uid=1000 pid=1190 comm="/usr/bin/gnome-shell" label="unconfined")
09:26:08 lbry: Daemon: 2023-02-08 09:26:08,452 INFO     lbry.torrent.tracker:190: Tracker: finished announcing 17 files to tracker.lbry.com:9252, 17 errors
09:19:12 NetworkManager: <info>  [1675865952.0988] dhcp4 (wlp3s0): state changed new lease, address=10.3.227.129
09:19:08 lbry: Daemon: 2023-02-08 09:19:08,398 INFO     lbry.torrent.tracker:190: Tracker: finished announcing 17 files to tracker.lbry.com:9252, 17 errors
09:18:34 systemd: systemd-hostnamed.service: Deactivated successfully.
09:18:04 dbus-daemon: [system] Successfully activated service 'org.freedesktop.hostname1'
09:18:04 systemd: Starting Hostname Service...
09:18:04 dbus-daemon: [system] Activating via systemd: service name='org.freedesktop.hostname1' unit='dbus-org.freedesktop.hostname1.service' requested by ':1.63' (uid=1000 pid=1586 comm="/usr/libexec/xdg-desktop-portal-gnome" label="unconfined")
09:17:58 lbry: Daemon: 2023-02-08 09:17:58,869 INFO     lbry.wallet.ledger:150: lbc_mainnet: added BlockHeightEvent(height=1307164, change=1) header blocks, final height 1307164
09:17:53 systemd: Started Tracker metadata extractor.
09:17:53 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.freedesktop.Tracker3.Miner.Extract'
09:17:53 systemd: Starting Tracker metadata extractor...
09:17:53 dbus-daemon: [session uid=1000 pid=1042] Activating via systemd: service name='org.freedesktop.Tracker3.Miner.Extract' unit='tracker-extract-3.service' requested by ':1.84' (uid=1000 pid=1605 comm="/usr/libexec/tracker-miner-fs-3" label="unconfined")
09:17:39 firefox-bin: ATTENTION: default value of option mesa_glthread overridden by environment.
09:17:37 systemd: Started app-flatpak-org.mozilla.firefox-9569.scope.
09:17:36 gnome-photos: Unable to fetch the list of available renderers: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name com.intel.dleyna-renderer was not provided by any .service files
09:17:36 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.Photos'
09:17:36 gnome-shell: Timelines with detached actors are not supported. <unnamed>[<Gjs_ui_search_ListSearchResult>:0x558c160a5f90] in animation of duration 200ms but not on stage.
09:17:36 dbus-daemon: [session uid=1000 pid=1042] Activating service name='org.gnome.Photos' requested by ':1.30' (uid=1000 pid=1190 comm="/usr/bin/gnome-shell" label="unconfined")
09:17:35 gnome-shell: Timelines with detached actors are not supported. <unnamed>[<Gjs_ui_search_ListSearchResult>:0x558c0f2b7830] in animation of duration 200ms but not on stage.
09:17:35 gnome-photos: Unable to fetch the list of available renderers: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name com.intel.dleyna-renderer was not provided by any .service files
09:17:35 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.Console'
09:17:35 nautilus: Connecting to org.freedesktop.Tracker3.Miner.Files
09:17:35 gnome-shell: Received error from D-Bus search provider org.gnome.Terminal.desktop: Gio.DBusError: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Object does not exist at path “/org/gnome/Terminal/SearchProvider”
09:17:35 systemd: Started GNOME Terminal Server.
09:17:35 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.Terminal'
09:17:35 systemd: Starting GNOME Terminal Server...
09:17:35 dbus-daemon: [session uid=1000 pid=1042] Activating service name='org.gnome.Epiphany.SearchProvider' requested by ':1.30' (uid=1000 pid=1190 comm="/usr/bin/gnome-shell" label="unconfined")
09:17:34 systemd: app-flatpak-org.mozilla.firefox-8536.scope: Consumed 1min 37.028s CPU time.
09:17:01 cron: pam_unix(cron:session): session closed for user root
09:17:01 kernel: (root) CMD (   cd / && run-parts --report /etc/cron.hourly)
09:17:01 cron: pam_unix(cron:session): session opened for user root(uid=0) by (uid=0)
09:16:48 lbry: Daemon: 2023-02-08 09:16:48,392 INFO     lbry.torrent.tracker:190: Tracker: finished announcing 17 files to tracker.lbry.com:9252, 17 errors
09:15:06 firefox-bin: ATTENTION: default value of option mesa_glthread overridden by environment.
09:15:04 gnome-photos: Unable to fetch the list of available renderers: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name com.intel.dleyna-renderer was not provided by any .service files
09:15:04 systemd: Started app-flatpak-org.mozilla.firefox-8536.scope.
09:15:04 gnome-shell: Received error from D-Bus search provider org.gnome.Terminal.desktop: Gio.DBusError: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Object does not exist at path “/org/gnome/Terminal/SearchProvider”
09:15:04 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.Photos'
09:15:04 gnome-shell: Received error from D-Bus search provider org.gnome.Terminal.desktop: Gio.DBusError: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Object does not exist at path “/org/gnome/Terminal/SearchProvider”
09:15:04 dbus-daemon: [session uid=1000 pid=1042] Activating service name='org.gnome.Photos' requested by ':1.30' (uid=1000 pid=1190 comm="/usr/bin/gnome-shell" label="unconfined")
09:15:04 gnome-photos: Unable to fetch the list of available renderers: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name com.intel.dleyna-renderer was not provided by any .service files
09:15:03 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.Photos'
09:15:03 gnome-shell: Received error from D-Bus search provider org.gnome.Terminal.desktop: Gio.DBusError: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Object does not exist at path “/org/gnome/Terminal/SearchProvider”
09:15:03 dbus-daemon: [session uid=1000 pid=1042] Activating service name='org.gnome.Photos' requested by ':1.30' (uid=1000 pid=1190 comm="/usr/bin/gnome-shell" label="unconfined")
09:15:02 gnome-photos: Unable to fetch the list of available renderers: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name com.intel.dleyna-renderer was not provided by any .service files
09:15:02 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.DiskUtility'
09:15:02 nautilus: Connecting to org.freedesktop.Tracker3.Miner.Files
09:15:02 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.Epiphany.SearchProvider'
09:15:02 gnome-shell: Received error from D-Bus search provider org.gnome.Terminal.desktop: Gio.DBusError: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Object does not exist at path “/org/gnome/Terminal/SearchProvider”
09:15:02 systemd: Started GNOME Terminal Server.
09:15:02 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.Terminal'
09:15:02 systemd: Starting GNOME Terminal Server...
09:15:02 dbus-daemon: [session uid=1000 pid=1042] Activating service name='org.gnome.Epiphany.SearchProvider' requested by ':1.30' (uid=1000 pid=1190 comm="/usr/bin/gnome-shell" label="unconfined")
09:15:00 systemd: app-flatpak-io.freetubeapp.FreeTube-8112.scope: Consumed 56.278s CPU time.
09:14:28 lbry: Daemon: 2023-02-08 09:14:28,342 INFO     lbry.torrent.tracker:190: Tracker: finished announcing 17 files to tracker.lbry.com:9252, 17 errors
09:14:04 gnome-shell: JS WARNING: [resource:///org/gnome/gjs/modules/core/overrides/Gio.js 287]: Too many arguments to method Gio.AsyncInitable.init_async: expected 3, got 4
09:13:47 cat: [47:0208/141347.389031:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 3 times!
09:13:40 systemd: Started app-flatpak-io.freetubeapp.FreeTube-8146.scope.
09:13:39 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.freedesktop.portal.Flatpak'
09:13:39 lbry: Function provided here: ui.js:2:4440665
09:13:39 systemd: Starting flatpak portal...
09:13:39 dbus-daemon: [session uid=1000 pid=1042] Activating via systemd: service name='org.freedesktop.portal.Flatpak' unit='flatpak-portal.service' requested by ':1.234' (uid=1000 pid=8121 comm="xdg-dbus-proxy --args=41" label="unconfined")
09:13:39 systemd: Started app-flatpak-io.freetubeapp.FreeTube-8112.scope.
09:13:39 lbry: Function provided here: ui.js:2:4440665
09:13:39 systemd: Started Application launched by gnome-shell.
09:13:39 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.Console'
09:13:39 gnome-photos: Unable to fetch the list of available renderers: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name com.intel.dleyna-renderer was not provided by any .service files
09:13:39 dbus-daemon: [session uid=1000 pid=1042] Activating service name='org.gnome.DiskUtility' requested by ':1.219' (uid=1000 pid=8005 comm="/usr/bin/nautilus --gapplication-service" label="unconfined")
09:13:38 nautilus: Connecting to org.freedesktop.Tracker3.Miner.Files
09:13:38 gnome-shell: Received error from D-Bus search provider org.gnome.Terminal.desktop: Gio.DBusError: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Object does not exist at path “/org/gnome/Terminal/SearchProvider”
09:13:38 systemd: Started GNOME Terminal Server.
09:13:38 dbus-daemon: [session uid=1000 pid=1042] Successfully activated service 'org.gnome.Terminal'
09:13:38 systemd: Starting GNOME Terminal Server...
09:13:38 dbus-daemon: [session uid=1000 pid=1042] Activating service name='org.gnome.Epiphany.SearchProvider' requested by ':1.30' (uid=1000 pid=1190 comm="/usr/bin/gnome-shell" label="unconfined")
09:13:18 lbry: Daemon: 2023-02-08 09:13:18,338 INFO     lbry.torrent.tracker:190: Tracker: finished announcing 17 files to tracker.lbry.com:9252, 17 errors
09:12:19 gnome-shell: JS WARNING: [resource:///org/gnome/gjs/modules/core/overrides/Gio.js 287]: Too many arguments to method Gio.AsyncInitable.init_async: expected 3, got 4
09:12:19 lbry: Daemon: 2023-02-08 09:12:19,984 WARNING  lbry.stream.managed_stream:244: connection was reset after sending browser 4194302 blob bytes
09:11:16 gnome-shell: Window manager warning: META_CURRENT_TIME used to choose focus window; focus window may not be correct.
09:11:14 systemd: app-flatpak-org.mozilla.firefox-3342.scope: Consumed 7min 59.492s CPU time.
09:10:58 lbry: Daemon: 2023-02-08 09:10:58,329 INFO     lbry.torrent.tracker:190: Tracker: finished announcing 17 files to tracker.lbry.com:9252, 17 errors
09:09:45 gnome-shell: libinput error: event6  - ETPS/2 Elantech Touchpad: WARNING: log rate limit exceeded (5 msgs per 24h). Discarding future messages.
09:08:38 lbry: Daemon: 2023-02-08 09:08:38,289 INFO     lbry.torrent.tracker:190: Tracker: finished announcing 17 files to tracker.lbry.com:9252, 17 errors
09:06:09 systemd: app-flatpak-org.mozilla.firefox-6882.scope: Consumed 1min 33.393s CPU time.
09:06:08 lbry: Daemon: 2023-02-08 09:06:08,489 INFO     lbry.torrent.tracker:190: Tracker: finished announcing 1 files to tracker.lbry.grin.io:9252, 0 errors
~~~
~~~
10:50:20 openvpn: Unable to retain capabilities
10:50:05 kernel: ACPI Error: Aborting method \_SB.PEP._DSM due to previous error (AE_NOT_FOUND) (20220331/psparse-529)
09:43:41 openvpn: event_wait : Interrupted system call (fd=-1,code=4)
08:49:14 systemd: Failed to start Update the local ESM caches.
08:49:07 gdm-session-wor: gkr-pam: couldn't unlock the login keyring.
08:49:07 kernel: Bluetooth: hci0: Malformed MSFT vendor event: 0x02
08:49:07 systemd: Failed to start casper-md5check Verify Live ISO checksums.
08:49:05 kernel: Serial bus multi instantiate pseudo device driver INT3515:00: error -ENXIO: Error requesting irq at index 1: -6
08:49:04 kernel: vso-autoupdate.service: Job vso-autoupdate.timer/start deleted to break ordering cycle starting with vso-autoupdate.service/start
~~~

**Comment by mirkobrombin** [comment]
This should be reported to GNOME if persist after Orchid.

**Comment by kbdharun** [comment]
> This should be reported to GNOME if persist after Orchid.

Tested this again on a VM running Orchid. I am still unable to replicate it. So I think it got resolved. (@gonzoknows if you still have this issue in Orchid, please let us know)
